test/tasks/test_gapic.py

```diff
@@ -189,7 +189,8 @@ def test_execute(self, exec_command, check_output):
             organization_name='google-cloud',
             package_metadata_yaml='/path/to/pmy.yaml',
             discovery_doc="/path/to/compute.v1.json",
-            toolkit_path='/path/to/toolkit'
+            toolkit_path='/path/to/toolkit',
+            root_dir='root_dir'
```

Comment:
1. A root dir should start with a slash
2. This doesn't seem to be influencing the commands below
3. This usage of an absolute path doesn't seem necessary for gapic generation; I think it's because I invoke gapic generation using the full path to the gapic config, so it's already an absolute path.

Reply:
1, 3: What if I want to run artman from a path relative to my current directory?
2: Fixed the test.

Reply:
1, 3: That also works too (currently for GAPIC)

Reply:
The problem is that artman resolves all filepaths in the config to be relative to the config filepath and assumes that the artman config file is at the highest (or equal) level directory compared to the files in the config. 
For example, in googleapis, for `google/pubsub/artman_pubsub.yaml`, all the pubsub API files are located at or below that directory level, like `google/pubsub/v1/pubsub.proto`.

However, in discovery-artifact-manager, the relevant discovery document is not in the same directory path as the artman config.
For example, for `gapic/google/compute/artman_compute.yaml`, the relevant discovery doc is at `discoveries/compute.v1.json`.

So there are two options:
1. this proposed PR, which allows specifying the base `discovery-artifact-manager` location
2. Removing the check in https://github.com/googleapis/artman/blob/master/artman/config/loader.py#L198, which would allow specifying the discovery_doc path as "../../discoveries/compute.v1.json" relative to the artman config filepath.